dali/dfuplus/dfuplus.cpp

```diff
@@ -351,7 +351,9 @@ int CDfuPlusHelper::doit()
     return 0;
 }
 
-bool CDfuPlusHelper::fixedSpray(const char* srcxml,const char* srcip,const char* srcfile,const MemoryBuffer &xmlbuf,const char* dstcluster,const char* dstname,const char *format, StringBuffer &retwuid, StringBuffer &except)
+bool CDfuPlusHelper::fixedSpray(const char* srcxml,const char* srcip,const char* srcfile,const char* srcplane,
```

Comment:
How does dstplane differ from dstcluster?

Reply:
The despray ESP service and the despray action require the dstplane.  I added this to spray for consistency.  I didn't want users to be confused by having to used dstcluster with spray and dstplane for despray.

Reply:
do you mean rename dstcluster to dstplane?  I agree dstplane makes more sense.  However, The FileSpray ESP service and dfuplus still uses dstcluster.  I think it better to change it every where in one go.  May be do this everywhere in a separate jira?

Reply:
not critical, for me because a 'plane' param is being introduced in same method that there's a 'cluster' based param, this is the time to change this one, but in general others can be tackled incrementally.